### ISSUE TITLE
Compliant with ISO 18004

### ISSUE THREAD

**Issue by YohanSciubukgian** [nayuki/QR-Code-generator#105]
You are mentioning ISO/IEC 18004 (2006) standard in your documentation (https://www.nayuki.io/page/qr-code-generator-library)
![image](https://user-images.githubusercontent.com/30290851/100883708-eec97280-34b0-11eb-9e00-ba74dba8bf8e.png)

Could you guarantee that your libraries are compliant with this standard ?
If yes, could you add it to the readme ?

**Comment by nayuki** [comment]
I linked to the ISO specification on my project web page because that is the reference document to learn the QR Code specification. But to be able to say or guarantee that my library is "compliant" with the standard is tricky for many reasons:
* I am not aware that the standard defines a set of compliance test cases for formal verification.
* The standard is a paid download and may require joining the ISO.
* I did not implement all parts of the standard: e.g., structured append, FNC, Micro QR Code, QR Code Model 1.
* Some parts of the standard are under-specified / ambiguous, such as the the penalty scoring algorithm.
* However, all the functionality that I have implemented is correct to the best of my knowledge - especially after many tests, thorough code review, and years of working with the same codebase.
